packages/desktop/src/renderer/app/components/environment-routes/environment-routes.component.html

```diff
@@ -405,7 +405,7 @@
                     *ngIf="
                       rulesDisablingResponseModes.includes(
                         data.activeRoute?.responseMode
-                      ) || !!data.activeRoute.streamingMode
+                      ) || data.activeRoute.streamingMode === 'BROADCAST'
```

Comment:
I initially disabled rules for both streaming modes, because, of this path parameter limitation. I wanted to enable rules for one-to-one after implementation of path parameters which does this PR now. So, responses for each client can be customized independently by using any means of request parameters.

Reply:
Thanks for the explanation, it makes sense :)